tests/retrieveBearerToken.test.js

```diff
@@ -0,0 +1,109 @@
+import { createMocks } from "node-mocks-http";
+import client from "next-auth/client";
+import { retrieve } from "../pages/api/id/[form]/bearer";
+import fetchMock from "jest-fetch-mock";
+
+jest.mock("next-auth/client");
+
+describe("Test bearer token retrieve api endpoint", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    fetchMock.enableMocks();
+  });
+
+  it("Shouldn't allow GET request without a valid form's id", async () => {
```

Comment:
How are you mocking the database here ?

Reply:
I did the db test but forgot it in a "stash tree". Will bring it back.

Reply:
I don't mean testing the database I mean mocking out the database connection